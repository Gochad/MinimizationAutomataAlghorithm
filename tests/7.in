0 1 2 3 4 5 6 7
0
2
a b
0 a 1
0 b 5
1 a 6
1 b 2
2 a 0 
2 b 2
3 a 2
3 b 6
4 a 7
4 b 5
5 a 2
5 b 6
6 a 6
6 b 4
7 a 6
7 b 2