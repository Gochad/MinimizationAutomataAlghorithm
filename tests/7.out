6 04 17 35 2 
0
2 
a b 
6 a 6
6 b 04
04 a 17
04 b 35
17 a 6
17 b 2
35 a 2
35 b 6
2 a 04
2 b 2
